src/shared/utils/mixpanel/mixpanel.utils.ts

```diff
@@ -26,19 +26,49 @@ export const trackAppletSave = ({
 }) => {
   if (!applet) return;
 
+  let itemCount: number = 0;
+  let phraseBuilderItemCount: number = 0;
+  let itemsIncludedInPhraseBuilders: number = 0;
   const itemTypes: ItemResponseType[] = [];
+
   (applet.activities ?? []).forEach((activity) => {
-    itemTypes.push(...(activity.items ?? []).map((item) => item.responseType));
+    const referencedItemNames = new Set<string>();
+    const items = activity.items ?? [];
+
+    for (const item of items) {
+      itemTypes.push(item.responseType);
+
+      if (item.responseType === ItemResponseType.PhrasalTemplate) {
+        phraseBuilderItemCount++;
+
+        for (const phrase of item.responseValues.phrases) {
+          for (const field of phrase.fields) {
+            if (field.type === 'item_response') {
+              referencedItemNames.add(field.itemName);
+            }
+          }
+        }
+      }
+    }
+
+    itemCount += items.length;
+    itemsIncludedInPhraseBuilders += referencedItemNames.size;
   });
   const uniqueItemTypes = new Set(itemTypes);
 
   const event: AppletSaveEvent = {
     action,
     [MixpanelProps.AppletId]: appletId,
     [MixpanelProps.ItemTypes]: [...uniqueItemTypes],
+    [MixpanelProps.ItemCount]: itemCount,
+    [MixpanelProps.PhraseBuilderItemCount]: phraseBuilderItemCount,
+    [MixpanelProps.ItemsIncludedInPhraseBuilders]: itemsIncludedInPhraseBuilders,
+    [MixpanelProps.AverageItemsPerPhraseBuilder]: phraseBuilderItemCount
+      ? itemsIncludedInPhraseBuilders / phraseBuilderItemCount
```

Comment:
This calculation can result in long decimals, which may not be desirable. We could do something like this instead

```suggestion
      ? parseFloat((itemsIncludedInPhraseBuilders / phraseBuilderItemCount).toFixed(2))
```